### DISCUSSION TITLE
Zoom Video Encoding

### DISCUSSION THREAD

**Discussion by radiant-forever** [AntonyCorbett/OnlyM#435] [Ideas]
Zoom now has built in video player for sharing video, that also reduces CPU/GPU needed.  Follow this link for details:

https://support.zoom.us/hc/en-us/articles/360051673592-Sharing-and-playing-a-video

An improved solution for those without access to OnlyM like macOS and Linux users.  More than half our AV team is now running macOS equipment.  To access the feature, share your media, and select the Advanced tab on the share options screen, then select Video, then browse for the video or song to stream.  It helps to put everything in a folder that is organized and ready to go, like the OnlyM Media folder.  No controls, overlays, or borders are shown for the video share using this feature.

For OnlyM, development question:

Is it possible to utilize Zooms SDK to take advantage of this zoom built in encoding straight from OnlyM?

**Comment by AntonyCorbett** [comment]
> Is it possible to utilize Zooms SDK to take advantage of this zoom built in encoding straight from OnlyM?

I don't know for sure - the Zoom API docs are currently baffling :)

**Comment by sircharlo** [comment]
You might be interested in an app I made called JWMMF. It's currently being used in hundreds of congregations; it allows to extract all meeting media automatically and convert it all into MP4 files (even the pictures) for easy Zoom sharing. 

Windows, macOS, Linux. Check it out on my profile.

**Reply by radiant-forever** [reply]
That’s great Charles, will check it out!
